src/components/SwatchPaletteWidget/SwatchPalette.js

```diff
@@ -1,556 +1,598 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { settings } from 'carbon-components';
+import {
+  black,
+  blue,
+  coolGray,
+  warmGray,
+  white,
+  yellow,
+  magenta,
+  orange,
+  teal,
+  purple,
+  red,
+  green,
+  gray,
+  cyan,
+} from '@carbon/colors';
+const convert = require('color-convert');
 
 const { prefix } = settings;
 
+const convertToCmyk = hex => {
+  //converts hex to cmyk => returns num array
+  const convertedHex = convert.hex.cmyk(hex);
+
+  // converts array items to string with cmyk letter
+  const stringCmykArr = convertedHex.map((i, index) => {
+    const stringNum = `${i.toString()}`;
+    let letterNum;
+    if (index === 0) {
+      letterNum = `c${stringNum}`;
+    } else if (index === 1) {
+      letterNum = `m${stringNum}`;
+    } else if (index === 2) {
+      letterNum = `y${stringNum}`;
+    } else {
+      letterNum = `k${stringNum}`;
+    }
+    return letterNum;
+  });
+
+  // converts cmyk array to one string
+  const cmykString = stringCmykArr.join(' ');
+  return cmykString;
+};
+
 const PALETTES = {
   black: {
     '100': {
-      hex: '#000000',
+      hex: `${black}`,
```

Comment:
Would this also work?

```suggestion
      hex: black,
```